sys/arm64/include/cherireg.h

```diff
@@ -169,4 +175,17 @@
 #define	CHERI_OTYPE_UNSEALED	(0l)
 #define	CHERI_OTYPE_SENTRY	(1l)
 
+/*
+ * Root compartment ID capablity for userspace.
+ *
+ * XXX: move these to sys/cheri/cherireg.h if another platform implements CIDs.
+ *
+ * XXX: a larger CID space is possible, but unr(9) only supports UINT_MAX
+ * so go with that for now.
+ */
+#define	CHERI_COMPARTMENT_ID_USERSPACE_PERMS	CHERI_PERMS_USERSPACE_CIDCAP
+#define	CHERI_COMPARTMENT_ID_USERSPACE_BASE	0x0
+#define	CHERI_COMPARTMENT_ID_USERSPACE_LENGTH	0xFFFFFFFF
```

Comment:
I'm going to change this to 0x8FFFFFFFFFFFFFFF and use unr64 to implement the allocator.